app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt

```diff
@@ -27,13 +27,18 @@ class AutocompleteQuickAddPopup(context: Context, url: String) : PopupWindow() {
         setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
 
         val button = view.findViewById<Button>(R.id.quick_add_autocomplete_button)
+
+        var URLAlreadyExists = false
```

Comment:
\[detekt\] INFO: \[style/NamingConventionViolation\] All names in the codebase should be matching the naming convention of the codebase\.